cmd/root.go

```diff
@@ -113,7 +112,6 @@ func Execute() {
 	}
 
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(err)
```

Comment:
this was double printing the error like so:
```
Error: failed to send first transaction
failed to send first transaction
```

Reply:
Thank you for finding this!